theano/gpuarray/dnn.py

```diff
@@ -3038,6 +3038,9 @@ def local_abstractconv_cudnn_graph(op, context_name, inputs, outputs):
     if op.unshared:
         return None
 
+    if isinstance(op.border_mode, tuple) and any(isinstance(p, tuple) for p in op.border_mode):
```

Comment:
Sorry, I didn't understand this. Could you explain in a little more detail? Which manual padding?

Reply:
I think that refers to the second part of my comment, where I suggested to use cuDNN with asymmetric padding. Depending on the padding values, you'd have to manually pad the input before you give it to the cuDNN op or manually crop the result of the output. That would work, but it might indeed use more memory and might also be slower than GpuCorrMM with custom padding.

Reply:
Got it, thanks